packages/external-integration/.eslintrc.json

```diff
@@ -1,4 +1,12 @@
 {
     "extends": ["../../.eslintrc.json"],
-    "ignorePatterns": ["!**/*"]
+    "ignorePatterns": ["!**/*"],
```

Comment:
Since we don't have `"ignorePatterns": ["**/*"]` in base, do we still need this line?

Reply:
It's mostly copy paste, should be able to remove this.